htdocs/bootstrap/css/custom-css.css

```diff
@@ -453,3 +453,28 @@ a.btn.btn-primary {
 .sweet-alert p {
   word-wrap: break-word;
 }
+
+/* Loader customizations */
+
+.loader-container {
+    height: 80vh;
+    width: 100%;
```

Comment:
why is one measured in vh but the other in percentage? I think the component would compose better if both were measured in percentages.

Reply:
When expressing the height as a percentage, the loading circle would be positioned immediately below the breadcrumbs, no matter the percentage value. To resolve this it seemed like either needed to express the height as a fixed amount of pixels, or as a relative height of the viewport. I figured the latter would be more consistent across monitors.

![loadernowww](https://user-images.githubusercontent.com/7550451/41428583-8748b7f0-6fd8-11e8-8e1e-42b933b378e3.png)

Reply:
why not add padding to the loader to ensure it doesn't hit up against the top? Measuring in vh means you can only use it for "page loading" and not for some sub-component of a page loading. I'd try adding some amount of padding measured in em (my gut tells me 3em would probably be fine...)

It also seems like you're trying to use flexbox just to center one item, which is overkill. You could use `margin-left: auto; margin-right: auto;` to do that going all the way back to browsers with CSS1 support.

(At that point, you might not even need the container at all, you could just make the loader a block component with centered block element with padding..)

Reply:
Sounds good! I'll give that a shot tomorrow.